Lord Forte Backlog.rb

```diff
@@ -276,8 +275,7 @@ def push(text)
   #--------------------------------------------------------------------------
   def create_log_contents(size)
     self.contents.dispose
-   # self.contents = Bitmap.new(width - padding * 2, [height - padding * 2, size * line_height].max)
-   self.contents = Bitmap.new(width, [height - padding * 2, size * line_height].max)
+    self.contents = Bitmap.new(width - padding * 2, [height - padding * 2, size * line_height].max)

```

Comment:
This removes the right-scroll icon and ensures the size of the content pane respects the actual available space.